rllib/algorithms/tests/test_callbacks.py

```diff
@@ -74,6 +83,29 @@ def setUpClass(cls):
     def tearDownClass(cls):
         ray.shutdown()
 
+    def test_on_init_and_checkpoint_loaded(self):
+        config = (
+            PGConfig()
```

Comment:
We are moving PG to rllib contrib. use PPO.

Reply:
done